src/constants/MarkdownFiles/posts/2025-06-15-gsoc-25-BishoyWadea-week02.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "GSoC ’25 Week 02 Update by Bishoy Wadea"
+excerpt: "Broken Calculator"
+category: "DEVELOPER NEWS"
+date: "2025-06-15"
+slug: "gsoc-25-BishoyWadea-week01"
+author: "Bishoy Wadea"
+description: "GSoC'25 Contributor at SugarLabs"
+tags: "gsoc25,sugarlabs,week02,BishoyWadea"
+image: "assets/Images/GSOC.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 01 Progress Report by Bishoy Wadea
+
+**Project:** [Broken Calculator](https://github.com/Bishoywadea/Four-Color-Map)  
+**Mentors:** [Ibiam Chihurumnaya](https://github.com/chimosky)  
+**Assisting Mentors:** [Walter Bender](https://github.com/walterbender/)  
+**Reporting Period:** 2025-06-08 - 2025-06-15 
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Define game features and core mechanics.
+- **Goal 2:** Design and plan a child-friendly, interactive game UI.
+- **Goal 3:** Implement the core game logic.
+
+---
+
+## This Week’s Achievements – *Broken Calculator Game*
+
+1. **Initial Setup & Core Functionality**
+   - Added starter files and project structure.
+   - Implemented basic game manager functionality to handle state, inputs, and equation validation.
+   - commit: [Initial Commit](https://github.com/Bishoywadea/BrokenCalculator/commit/initial)
```

Comment:
There are a lot of broken links in your blogpost, such as this one.

Please check all your links before making a PR.

![Screenshot from 2025-06-15 08-39-23](https://github.com/user-attachments/assets/8d60195b-ca38-4c30-9a88-2f057882f690)